packages/forms-web-app/__tests__/unit/controllers/examination/file-upload/handlers.test.js

```diff
@@ -0,0 +1,108 @@
+const {
+	deleteHandler,
+	uploadHandler
+} = require('../../../../../src/controllers/examination/file-upload/handlers');
+const {
+	deleteFileOnDisk
+} = require('../../../../../src/controllers/examination/file-upload/fileManagement');
+const {
+	deleteFileInSession,
+	getUploadedFilesFromSession
+} = require('../../../../../src/controllers/examination/file-upload/fileSessionManagement');
+const {
+	noFileSelected,
+	moreThanXAmountFiles,
+	handleMultipleFileUploadsWithErrors
+} = require('../../../../../src/controllers/examination/file-upload/fileValidation');
+
+jest.mock('../../../../../src/controllers/examination/file-upload/fileManagement', () => ({
+	deleteFileOnDisk: jest.fn()
+}));
+jest.mock('../../../../../src/controllers/examination/file-upload/fileSessionManagement', () => ({
+	deleteFileInSession: jest.fn(),
+	getUploadedFilesFromSession: jest.fn()
+}));
+jest.mock('../../../../../src/controllers/examination/file-upload/fileValidation', () => ({
+	noFileSelected: jest.fn(),
+	moreThanXAmountFiles: jest.fn(),
+	handleMultipleFileUploadsWithErrors: jest.fn()
+}));
+describe('controllers/examination/file-upload/fileValidation', () => {
+	describe('#deleteHandler', () => {
+		describe('When deleting a file', () => {
```

Comment:
this is just testing mocks are called. But the coverage will dip below threshold.